chunjun-e2e/src/test/java/com/dtstack/chunjun/connector/entity/LaunchCommandBuilder.java

```diff
@@ -63,13 +70,13 @@ public LaunchCommandBuilder withFlinkConfDir(String flinkConfDir) {
 
     public LaunchCommandBuilder withFlinkCustomConf(Map<String, Object> properties) {
         commands.add("-confProp");
-        commands.add(GsonUtil.GSON.toJson(properties));
+        commands.add(new GsonBuilder().create().toJson(properties));
         return this;
     }
 
     public LaunchCommandBuilder withAddJar(List<String> path) {
         commands.add("-addjar");
-        commands.add(GsonUtil.GSON.toJson(path));
+        commands.add(new GsonBuilder().create().toJson(path));
```

Comment:
Why use new Gson rather than GsonUtil.GSON?

Reply:
At this time, because GsonUtil uses prettyjson by default, when using the shell to submit, we cannot appear \r\n json data.

Reply:
The json we use must be one line of data.